src/app/case-studies/_ui/index.tsx

```diff
@@ -0,0 +1,75 @@
+import Link from 'next/link';
```

Comment:
The Next docs recommend using [private folders](https://nextjs.org/docs/app/getting-started/project-structure#private-folders) to share code like this between pages.